decidim-blogs/spec/cells/decidim/blogs/post_g_cell_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim::Blogs
+  describe PostGCell, type: :cell do
+    controller Decidim::ApplicationController
+    include Decidim::TranslatableAttributes
+
+    subject { cell("decidim/blogs/post_g", model).call }
+
+    let(:model) { create(:post) }
+
+    context "when rendering" do
+      let(:show_space) { false }
```

Comment:
Is this method used in the test? Maybe it could be removed.

In case to be used it can be passed to the cell as option
```
    subject { cell("decidim/blogs/post_g", model, show_space:).call }
```
And check that the participatory space name the post belongs to is not displayed. Keep in mind that this option only applies to card-g cells of resources of components belonging to a participatory space, in the card-g of an assembly, for example, which is a participatory space itself this option is ignored